apps/src/code-studio/levels/codeStudioLevels.js

```diff
@@ -0,0 +1,33 @@
+let registeredGetResult = null;
+
+/**
+ * A number of our levels provide a function that can be used to get results.
+ * Previously this was just injected into the global namespace, making it
+ * difficult to track. This makes both the registration and the usage more explicit.
+ */
+export function registerGetResult(getResultFunction=basicGetResult) {
+  if (registeredGetResult) {
+    console.error('already have a getResult function');
```

Comment:
In this error case you are still overwriting `registeredGetResult`. Is this the intended behavior?